project/src/index.tsx

```diff
@@ -2,12 +2,16 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './components/app/app';
 
+const Settings = {
+  offersCount: 313,
+} as const;
+
 const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement,
 );
 
 root.render(
   <React.StrictMode>
-    <App />
+    <App offersCount = {Settings.offersCount} />
```

Comment:
Форматируем только так
```suggestion
    <App offersCount={Settings.offersCount} />
```